infrastructure/backups/restore.sh

```diff
@@ -31,12 +31,16 @@ for i in "$@"; do
     LABEL="${i#*=}"
     shift
     ;;
+  --run_cleanup=*)
```

Comment:
Same here: what if we just separate the clean completely to its own process and keep both backup and restore single-purpose?